cabot/metricsapp/forms/grafana_elastic.py

```diff
@@ -48,6 +49,7 @@ class Meta:
             'warning_value',
             'high_alert_importance',
             'high_alert_value',
+            'consecutive_failures',
```

Comment:
one more place to add this: https://github.com/Affirm/cabot/blob/master/cabot/metricsapp/forms/elastic.py#L27 (form for the admin site)

Reply:
Done